packages/earth-map/src/pages/change-email/component.tsx

```diff
@@ -56,7 +56,7 @@ export default function ChangeEmailComponent() {
       } catch (e) {
         console.error(e);
       } finally {
-        replace('/earth');
+        replace('/');
```

Comment:
The user will see spinny globe. We want this?

Reply:
dunno, page is change-email page is blank at the moment, will redo once we implement the page